### ISSUE TITLE
Sometimes "validate cache" takes a very long time

### ISSUE THREAD

**Issue by certik** [prefix-dev/pixi#2325]
Here is an example of 10 minutes and going:

![Image](https://github.com/user-attachments/assets/d442f537-cf38-4e3b-a5fa-dc588ee054fd)

I am guessing it is downloading something from the internet? Maybe the download is slow for some reason, hard to tell from the output.

To fix this issue, let's add the download speed and how much is left to download in MB, and adjust the progress bar to not be stuck at 99%.

**Comment by certik** [comment]
Yes, it finished in about 12 minutes. As a user I would expect this to finish in about 5s or so. The `.pixi` directory is about 1GB. I am pretty sure it should be possible to unpack 1GB of data under 12 minutes. :)

**Comment by wolfv** [comment]
I was just wondering if it is a "deadlock" situation and would resolve more quickly if you restart the process.

It can sometimes take long(ish) to validate lots of small files. 
I think we could think of an option to turn off the SHA validation of the cache.

**Comment by certik** [comment]
I think the main thing to fix is the progress bar: it takes about 2 minutes to progress to 99%, and then 10 minutes for the last 1%. Rather, if there is a job for 12 minutes, the the progress bar should go at a uniform rate, so that the user can make a coffee and finish it. I was glued to the screen ready to work, since 1% should finish anytime. :)
